runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -4,6 +4,7 @@
 import com.google.protobuf.Message;
 import io.micrometer.core.instrument.Timer;
 import lombok.Getter;
+import lombok.Setter;
```

Comment:
The commit for queue open has been merged and these are related changes can be removed.